### PR TITLE
made Symfony compatible with both Twig 1.x and 2.x

### PR DESCRIPTION
**Pull request by fabpot**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | n/a |
| License | MIT |
| Doc PR | n/a |

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -43,6 +46,8 @@ before_install:
     - if [ "$TRAVIS_BRANCH" = "master" ]; then export COMPOSER_ROOT_VERSION=dev-master; else export COMPOSER_ROOT_VERSION="$TRAVIS_BRANCH".x-dev; fi;
 
 install:
+    - if [ "$twig2" = "yes" ]; then sed -i 's/~1.20|~2.0/~2.0@dev/g' composer.json; composer update --prefer-source; fi;
+    - if [ "$twig2" = "no" ]; then sed -i 's/~1.20|~2.0/~1.20/g' composer.json; composer update --prefer-source; fi;
```
Comment (stof) [inline .travis.yml]: we should have some builds running with unmodified deps IMO (deps=high and deps=low will already cover 2.0 and 1.20 btw)

Comment (fabpot) [inline .travis.yml]: indeed, deps is all we need.

**Comment by stof** [comment]
@fabpot please check the test which is failing on 3 different jobs in the tests of the special Twig filesystem loader

**Comment by fabpot** [comment]
@stof I'm aware of that failing test and I see why it fails. Patch coming soon.

**Comment by fabpot** [comment]
Now that all builds fail because of the deprecation notices and because fixing the remaining issue is not easy, I'm going to merge this one to reduce the noise.

**Pull request merged**